Source/Make.package

```diff
@@ -43,6 +44,7 @@ CEXE_sources += PeleLMeX_Diagnostics.cpp
 CEXE_sources += PeleLMeX_FlowController.cpp
 CEXE_sources += PeleLMeX_DeriveUserDefined.cpp
 CEXE_sources += PeleLMeX_BPatch.cpp
+CEXE_sources += PeleLMeX_PatchFlowVariables.cpp
```

Comment:
You'll also have to modify `CMake/BuildPeleExe.cmake` fof the CMake build system to work.

Reply:
Done